src/components/models/EntityTypes.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * @author aramsey
+ *
+ * Data resource types.  Any means it can be a file or a folder.
+ */
+export default {
+    FOLDER: "folder",
+    FILE: "file",
+    ANY: "any",
+};
```

Comment:
Minor- since you have used the term `resource` to reference files / folders in most of the code related to data listing, we can name this as `ResourceType` just for uniformity.

Reply:
Ah, good point.  I forget now where I got `EntityType` from, but `ResourceType` makes a lot more sense.